.rubocop.yml

```diff
@@ -32,6 +32,11 @@ Metrics/MethodLength:
   Exclude:
     - 'spec/**/*' # not a big deal in spec helper methods
 
+Metrics/ClassLength:
+  Max: 100
+  Exclude:
+    - 'app/models/**/*' #fat models and slim controllers principle
```

Comment:
We don't follow fat models, we want both controllers and model to be slim as possible